operator/src/main/resources/application.properties

```diff
@@ -2,3 +2,6 @@ quarkus.operator-sdk.crd.apply=true
 quarkus.operator-sdk.generate-csv=true
 quarkus.container-image.builder=jib
 quarkus.operator-sdk.crd.validate=false
+
+# Operator config
+operator.operand.image=quay.io/keycloak/keycloak-x:latest
```

Comment:
I would suggest another naming, as every element created by the operator are operands .... so probably operator.operand.keycloakImage could be clearer ? wdyt ?

Reply:
Renamed to `operator.keycloak.image`. WDYT?